centreon/packages/ui/cypress/support/commands.ts

```diff
@@ -0,0 +1,19 @@
+import { mount } from 'cypress/react18';
+import '@testing-library/cypress/add-commands';
+
+Cypress.Commands.add('mount', ({ Component, options }: MountProps) => {
+    return mount(Component, options);
+  });
+  
+  interface MountProps {
+    Component: JSX.Element;
+    options?: object;
+  }
+
+declare global {
+    namespace Cypress {
+      interface Chainable {
+        mount: ({ Component, options = {} }: MountProps) => Cypress.Chainable;
+      }
+    }
+  }
```

Comment:
can the type definitions be added to `tsconfig.json` ?

Reply:
I think it does not work for cypress

Reply:
it can be added to the `typeRoots` in the `tsconfig.json`

the `tsconfig.json` does not include them atm.
```
"typeRoots": [
      "./types",
      "./node_modules/@types"
    ],
``` 

adding `"./node_modules/@cypress"` should work for `cy`, the `testing-library` ones should be looked into as they seem to be hoisted and thus out of reach for typeRoots